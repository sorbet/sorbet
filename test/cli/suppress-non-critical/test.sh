#!/bin/bash
main/sorbet --silence-dev-message -e '!' --suppress-non-critical 2>&1
echo $?
