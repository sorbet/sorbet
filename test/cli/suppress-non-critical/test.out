-e:1: missing arg to "!" operator https://srb.help/2001
     1 |!
        ^
Errors: 1
0
